docs/ARCHITECTURE.md

```diff
@@ -1,7 +1,7 @@
 ## Architecture
 
 ### High-level overview
-![Draft Architecture](https://github.com/18F/identity-idp/raw/amos/feature/arch_docs_r1/docs/diagrams/draft-architecture-overview.png)
+![Draft architecture](https://raw.githubusercontent.com/18F/identity-idp/master/docs/diagrams/draft-architecture-overview.png)
```

Comment:
well I guess since it's already in the docs folder, just `diagrams/draft-architecture-overview.png`